src/main/java/pika/command/Command.java

```diff
@@ -0,0 +1,45 @@
+package pika.command;
+
+import java.io.IOException;
+
+import pika.exception.PikaException;
+import pika.ui.Storage;
+import pika.ui.TaskList;
+import pika.ui.Ui;
+
+/**
+ * Base Command Class for the other Command classes
+ */
+public class Command {
+    private final boolean isRunning;
+
+    /**
+     * Constructor for the command class
+     *
+     * @param isRunning boolean to indicate if the bot still runs after this function
+     */
+    public Command(boolean isRunning) {
+        this.isRunning = isRunning;
+    }
+
+    /**
+     * Checks if the bot still runs after this command
+     */
+    public boolean isRunning() {
+        return this.isRunning;
+    }
+
+    /**
+     * Executes the command
+     *
+     * @param taskList The current list of tasks
+     * @param ui       The current Ui
+     * @param storage  The current storage class to handle the txt file
```

Comment:
The return variable is missing from the javadoc comment.
```java
* @return ...
```

Reply:
Thanks for the catch! I forgot to update when i changed all of those to return a string instead of just printing it out